### ISSUE TITLE
-Wunused-result in fs/btrfs/volumes.c

### ISSUE THREAD

**Issue by nickdesaulniers** [ClangBuiltLinux/linux#875]
```
fs/btrfs/volumes.c:7338:3: warning: ignoring return value of function declared with 'warn_unused_result' attribute [-Wunused-result]
                write_one_page(page);
```

via -next allyesconfig

**Comment by nathanchance** [comment]
https://lore.kernel.org/lkml/20200212160341.GL2902@suse.cz/

**Comment by nathanchance** [comment]
Fixed in next-20200223.
